schema/va-spec/aac-2017/diagnostic-study-statement-profile-source.yaml

```
@@ -20,11 +20,79 @@ $defs:
       disease (diagnostic exclusion criterion) - based on interpretation of the study's
       results.
     allOf:
-    - $ref: "/ga4gh/schema/va-spec/1.0.0-ballot.2025-03.1/base/json/Statement"
+    - $ref: "/ga4gh/schema/va-spec/1.0.0-ballot.2025-03.2/base/json/Statement"
     - properties:
         proposition:
-          $ref: "/ga4gh/schema/va-spec/1.0.0-ballot.2025-03.1/base/json/VariantDiagnosticProposition"
+          $ref: "/ga4gh/schema/va-spec/1.0.0-ballot.2025-03.2/base/json/VariantDiagnosticProposition"
           description: >-
             A proposition about a diagnostic association between a variant and condition, for which the study provides
             evidence. The validity of this proposition, and the level of confidence/evidence supporting it,
             may be assessed and reported by the Statement.
+        strength:
+          description: >-
+            The strength of support that the Statement is determined to provide for or against the Diagnostic Proposition
+            for the assessed variant, based on the curation and reporting conventions of the AMP/ASCO/CAP (AAC) 2017
+            Guidelines. Note that the enumerated value set here is bound to the ‘code’ field of the Coding object that
+            is nested inside a MappableConcept's primary coding. Conditional requirement: Strength is evaluated relative to
+            the direction indicated by the 'direction' attribute. If direction is either 'supports' or 'disputes', the
+            'strength' attribute is required.  If the direction is 'neutral', then the 'strength' attribute is not allowed.
```

```suggestion
            The strength of support that the Statement is determined to provide for or against the Diagnostic Proposition
            for the assessed variant, based on the curation and reporting conventions of the AMP/ASCO/CAP (AAC) 2017
            Guidelines.
```

We traditionally provide implementation guidance in the docs, not in the schema field descriptions. This description also incorporates logic that we could encode directly as JSON Schema.